src/cbapi/psc/threathunter/query.py

```diff
@@ -379,20 +400,21 @@ def _submit(self):
             raise ApiError("Query already submitted: token {0}".format(self._query_token))
 
         args = self._get_query_parameters()
-        self._validate(args)
+        #self._validate(args)
+
+        url = "/api/investigate/v2/orgs/{}/processes/search_jobs".format(self._cb.credentials.org_key)
+        query_start = self._cb.post_object(url, body=args)
 
-        url = "/threathunter/search/v1/orgs/{}/processes/search_jobs".format(self._cb.credentials.org_key)
-        query_start = self._cb.post_object(url, body={"search_params": args})
+        self._query_token = query_start.json().get("job_id")
 
-        self._query_token = query_start.json().get("query_id")
         self._timed_out = False
         self._submit_time = time.time() * 1000
 
     def _still_querying(self):
         if not self._query_token:
             self._submit()
 
-        status_url = "/threathunter/search/v1/orgs/{}/processes/search_jobs/{}".format(
+        status_url = "/api/investigate/v1/orgs/{}/processes/search_jobs/{}".format(
```

Comment:
This is correct, there is no v2 API for search job status because the contract didn't need to change for that route. We may choose to introduce a v2 in the future but for now, this is good.